lbrynet/core/utils.py

```diff
@@ -9,6 +9,8 @@
 
 import pkg_resources
 
+from twisted.internet import reactor
```

Comment:
I think this should probably be in the `call_later` function, so that importing lbrynet.core.utils can't inadvertently configure an unconfigured reactor if it's imported early